spec/system/staff/viewing_incidents_spec.rb

```diff
@@ -79,70 +79,86 @@
     it 'starts out navigating by month' do
       visit incidents_url
       expect(page).to have_selector 'h2',
-        text: 'Monday, January 1 — Wednesday, January 31'
+                                    text: 'Monday, January 1 — Wednesday, January 31'
     end
     it 'allows going to the previous month' do
       visit incidents_url
       click_button '← Previous month'
       expect(page).to have_selector 'h2',
-        text: 'Friday, December 1 — Sunday, December 31'
+                                    text: 'Friday, December 1 — Sunday, December 31'
     end
     it 'allows going to the next month' do
       visit incidents_url
       click_button 'Next month →'
       expect(page).to have_selector 'h2',
-        text: 'Thursday, February 1 — Wednesday, February 28'
+                                    text: 'Thursday, February 1 — Wednesday, February 28'
     end
     it 'allows navigating by week' do
       visit incidents_url
       click_button 'View single week'
       expect(page).to have_selector 'h2',
-        text: 'Sunday, December 31 — Saturday, January 6'
+                                    text: 'Sunday, December 31 — Saturday, January 6'
     end
     context 'navigating by week' do
       it 'goes to the first week in the month, not the current week' do
         Timecop.freeze Date.new(2018, 1, 7) do
           visit incidents_url
           click_button 'View single week'
           expect(page).not_to have_selector 'h2',
-            text: 'Sunday, January 7 — Saturday, January 13'
+                                            text: 'Sunday, January 7 — Saturday, January 13'
           expect(page).to have_selector 'h2',
-            text: 'Sunday, December 31 — Saturday, January 6'
+                                        text: 'Sunday, December 31 — Saturday, January 6'
         end
       end
     end
     it 'allows going from week mode back to month mode' do
       visit incidents_url(mode: 'week')
       expect(page).to have_selector 'h2',
-        text: 'Sunday, December 31 — Saturday, January 6'
+                                    text: 'Sunday, December 31 — Saturday, January 6'
       click_button 'View for whole month'
       # Note that it doesn't go back to January,
       # because the first day of the week is in December.
       expect(page).to have_selector 'h2',
-        text: 'Friday, December 1 — Sunday, December 31'
+                                    text: 'Friday, December 1 — Sunday, December 31'
```

Comment:
ew... why the massive indentation? Is this a rubocop thing?

Reply:
Yes, should I remove it?

Reply:
Ok